### ISSUE TITLE
Redirect ignores Alpine -context parameter

### ISSUE THREAD

**Issue by patbaumgartner** [DependencyTrack/dependency-track#665]
### Current Behavior:
Having the Alpine parameter set on the Commandline _-context /dtrack_ in the previous version of DT and now migrating to 3.8 leads to a redirect to _/login_ without taking the _context-path_ into account.

### Steps to Reproduce:
``` 
java $JAVA_OPTIONS -DdependencyTrack.logging.level=$LOGGING_LEVEL -jar dependency-track-embedded.war -context /dtrack
```

### Expected Behavior:
I would expect that _-context_ is taken into account when redirects are made. This seems to be a regression because we had previously also problems with the _-context_ parameter.

### Environment:
  - Dependency-Track Version: 3.8
  - Distribution: Executable WAR
  - Browser: Chrome

### Additional Information:

Running it locally (_java -jar dependency-track-embedded.war -context=/dtrack_) shows that the _-context_ path is totally ignored. Everything is directly served via _/_ like  _/dashboards /projects /licenses_ and so on. 

This might be a regression that came in with Alpine 1.8.0-SNAPSHOT.

**Comment by stevespringett** [comment]
This is a known limitation of the UI at the moment. The context works for the server's APIs, but not the UI at this time. If you need more control over UI content and other things (TLS, etc), you can optionally deploy the UI independently of the server.

**Comment by patbaumgartner** [comment]
Thank you so much for your super fast feedback. DT is deployed behind an NGINX with several other apps. Not sure if I want to face other issues when using the frontend and another backend deployment. Probably it's better to wait a bit more. Not sure how I can contribute to that issue.

**Comment by stevespringett** [comment]
Closing. Fixed in v4.0 - to be released soon.
